src/instructlab/model/train.py

```diff
@@ -378,6 +378,12 @@ def clickpath_setup(is_dir: bool) -> click.Path:
     is_flag=True,
     help="Print serving engine logs during phased training checkpoint evaluation.",
 )
+@click.option(
```

Comment:
(Observation) In case other reviewers wonder how this even works since the argument is not passed into the `train` function: it's implicitly passed through `ctx.params` below. I kinda hate this approach (we better be more explicit here and avoid `ctx.params`) but this is not the issue of the PR. So just putting it here for other confused reviewers. :)

Reply:
I don't like this either. If our inputs to `train` were finalized we could pass function variables directly, but since the number of options changes frequently this is fairly convenient. 

The bigger thing to solve is the ballooning number of required options for this function. Is there a nicer way to scale this out than having tons of decorators?

Reply:
Decorators are kinda fine... They are almost declarative (so just "data") I don't think we should proliferate **required** options though.